apps/02RockPaperScissors.js

```diff
@@ -7,10 +7,55 @@ prompt.start();
 
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+    // Spec 3: Input scrubbing
+    // We're going to change the inputs to lower case.
+    hand1.toLowerCase();
+    hand2.toLowerCase();
+
+    // Spec 1: Is it a tie?
+    if (hand1 === hand2) {
+      return "It's a tie!";
+    }
+    // Now that we know it's not a tie:
+    // Spec 2: Who is the winner?
+    // First we'll determine the winner if hand1 is rock.
+    else  if (hand1 === 'rock'){
+      if (hand2 === 'scissors') {
+        return "Hand one wins!";
+      }
+      // We already know the other hand isn't rock, otherwise it would be
+      // a tie. Therefore, the other hand must be paper.
+      else {
+        return "Hand two wins!";
+      }
+    }
+    // Next, we'll determine the winner if hand1 is paper.
+    else if(hand1 === 'paper') {
+      if (hand2 === 'rock') {
+        return "Hand one wins!";
+      }
+      else {
+        return "Hand two wins!";
+      }
+    }
+    // Next, we'll determine the winner if hand1 is scissors.
+    else if (hand1 === 'scissors'){
+      if (hand2 === 'paper') {
+        return "Hand one wins!";
+      }
+      else {
+        return "Hand two wins!";
+      }
+    }
```

Comment:
Good if/else tree! You can further minimize the tree by exploring the logic operators:

 and ( && )  or ( || )

Example:

```javascript
else if ( 
        (hand1==='scissors' && hand2==='paper') || 
        (hand1=== 'paper' && hand2==='rock') || 
        (hand1==='rock' && hand2==='scissors')
        )
        { 
        return "Hand one wins!"; 
        }
```